extensions/resteasy-reactive/quarkus-resteasy-reactive/deployment/src/test/java/io/quarkus/resteasy/reactive/server/test/kotlin/BlockingCoroutinesTest.java

```diff
@@ -0,0 +1,43 @@
+package io.quarkus.resteasy.reactive.server.test.kotlin;
+
+import static org.junit.jupiter.api.Assertions.fail;
+
+import java.util.function.Supplier;
+
+import javax.ws.rs.Path;
+
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.spec.JavaArchive;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+import io.quarkus.test.QuarkusUnitTest;
+import io.smallrye.common.annotation.Blocking;
+
+public class BlockingCoroutinesTest {
+
+    @RegisterExtension
+    static QuarkusUnitTest test = new QuarkusUnitTest()
+            .setArchiveProducer(new Supplier<JavaArchive>() {
+                @Override
+                public JavaArchive get() {
+                    return ShrinkWrap.create(JavaArchive.class)
+                            .addClasses(TestEndpoint.class);
+                }
+            }).setExpectedException(IllegalStateException.class);
+
+    @Test
+    public void test() {
+        fail("Should never be called");
+    }
+
+    @Path("test")
+    public static class TestEndpoint {
+
+        @Path("hello")
+        @Blocking
+        public Object hello(kotlin.coroutines.Continuation<?> cont) {
```

Comment:
I would rather avoid having Kotlin dependency in this module.

Reply:
Even in the tests?

Reply:
Yes, even in the tests. That will complexity the build pipeline.

Reply:
Okay, I will then move this test to the integration tests module

Reply:
I added a new module for these tests